tox.ini

```diff
@@ -18,7 +18,7 @@ commands = coverage run --parallel -m pytest {posargs}
 
 [testenv:py36]
 deps = -rdev-requirements.txt
-commands = coverage run --parallel -m pytest {posargs}
+commands = coverage run --parallel -m pytest {posargs} tests/_test_annotations.py
```

Comment:
The `_test_` approach is not ideal because it relies on using tox to run the tests.

Please use the approach I took in https://github.com/hynek/prometheus_async/blob/d82f527c905cd581411007ccdbdaca396f0bad77/conftest.py#L23-L24 instead.